requirements.txt

```diff
@@ -0,0 +1,8 @@
+astropy==4.0.1.post1
+jax==0.1.76
+jaxlib
```

Comment:
hummm maybe we need a specific version for jaxlib... or actually, maybe we shouldn't list jaxlib here.... because it requires a specific install. If the user has a jax requirement, it's up to them to install an appropriate jaxlib separately

Reply:
I agree, I removed jaxlib from the requirements, I'll specify that a specific install of jaxlib is needed in the readme